index.js

```diff
@@ -742,8 +733,6 @@ for (const i in httpMethods) {
   const m = httpMethods[i]
   const methodName = m.toLowerCase()
 
-  if (Router.prototype[methodName]) throw new Error('Method already exists: ' + methodName)
```

Comment:
Uncovered branch.

I think `httpMethods` should be a trusted source: if it contains duplicates, the tests will fail in one way or another.